packages/fuels/tests/providers.rs

```diff
@@ -308,6 +311,7 @@ async fn contract_deployment_respects_maturity() -> Result<()> {
     Ok(())
 }
 
+#[ignore]
```

Comment:
?

Reply:
Forgot to reenable it.

Reply:
Here we test all the remaining gas is forwarded to the call instruction if nothing else was specified by call params. 
The test is failing right now because we check :
`gas_forwarded` (gas that was automatically forwarded to CALL) + `gas_used_by_script` = `gas_limit` (set with TxParams)

I'm wondering, was this test was malformed? `gas_used_by_script` also includes gas used by CALL. The correct was to calculate it would be:
`gas_forwarded`+ (`gas_used_by_script` - `gas_used_by_call`) = `gas_limit`
where `gas_used_by_call` is impossible to retrieve afaik.

Reply:
I had a look. It seems that `gas_used_by_script` is a `static` cost of the script (maybe cost for bytecode size, witnesses, etc.). Here are the numbers for the non-experimental case:

```
    gas_used_by_script = 364
    gas_forwarded_to_call = 225_519
    total_gas_used = 796
```
`total_gas_used` should be the `static` part plus the call cost. The calculation 
`gas_used_by_script` + `gas_forwarded` = `gas_limit` makes sense. 

We should change the name of `gas_used_by_script` to reflect the fact that it is not the total cost.